mappers/tourist-attraction.js

```diff
@@ -50,7 +50,7 @@ export default function mapTouristAttractions(records, translations) {
       if (type) {
         publicG.add(sym(attractionUri), SCHEMA('keywords'), sym(type));
       } else {
-        console.error(`Cannot map subtype value '${record['sub_type']}' for record ${recordId}`);
+        errorLogger('subtype', record['sub_type'], recordId);
```

Comment:
Pass the same field name as used to lookup the value as first argument. I know it was different before, but old code is wrong.